synapse/lib/coro.py

```diff
@@ -203,3 +208,29 @@ def execspawn():
     except (asyncio.CancelledError, asyncio.TimeoutError):
         proc.terminate()
         raise
+
+# shared process pool
+if multiprocessing.current_process().name == 'MainProcess':
+    mpctx = multiprocessing.get_context('forkserver')
+    max_workers = os.getenv('SYN_FORKED_WORKERS')
```

Comment:
Maybe we could have a default max of 4 or so?  Might prevent instances of big boxes from firing too many without specified values.

Reply:
:)
updated